d2bs/kolbot/libs/bots/Wakka.js

```diff
@@ -7,16 +7,16 @@
 var stopLvl = 99;
 
 function Wakka() {
-	var i, safeTP, portal, vizClear, seisClear, infClear, tick, diablo,
-		timeout = 1, // minutes
+	let i, safeTP, portal, vizClear, seisClear, infClear, tick, diablo,
+		timeout = Config.Wakka.Wait,
 		minDist = 50,
 		maxDist = 80,
 		leaderUnit = null,
 		leaderPartyUnit = null,
 		leader = "";
 
 	function autoLeaderDetect(destination) { // autoleader by Ethic
-		var solofail, suspect;
+		let solofail, suspect;
```

Comment:
replacing var with let was really necessary?

Reply:
any replaced var is good